### DISCUSSION TITLE
How to remove a module from the panel? (E.G the system tray from the bar)

### DISCUSSION THREAD

**Discussion by nibiritis** [Jas-SinghFSU/HyprPanel#323] [Q&A]
I want to remove/hide the system tray from my bar, but I could not find any settings in the settings menu.

**Comment by anotherhadi** [comment]
> I want to remove/hide the system tray from my bar, but I could not find any settings in the settings menu.

You should find where to edit this in the docs: https://hyprpanel.com/configuration/modules.html
Settings > Bar > Layouts
